lib/pure/math.nim

```diff
@@ -1182,6 +1182,14 @@ func lcm*[T](x, y: T): T =
 
   x div gcd(x, y) * y
 
+func clamp*[T](val: T, rng: Slice[T]): T {.since: (1, 5).}=
+  ## Like `system.clamp` but takes a slice so you can easily clamp within a range.
```

Comment:
* cross-reference from system.clamp
(maybe we should also deprecate system.clamp, it's too niche for system)